platform/linux-generic/odp_timer.c

```diff
@@ -1471,16 +1453,15 @@ void odp_timer_pool_destroy(odp_timer_pool_t tpid)
 
 uint64_t odp_timer_current_tick(odp_timer_pool_t tpid)
 {
-	timer_pool_t *tp = timer_pool_from_hdl(tpid);
+	(void)tpid;
```

Comment:
`ODP_UNUSED` would save one line.

Reply:
Cast to void is used with other inline functions. This function moves into inline header in the next patch.